frontend/src/pages/Vulnerability/Vulnerability.tsx

```diff
@@ -244,6 +244,44 @@ export const Vulnerability: React.FC = () => {
                       </Table>
                     </div>
                   )}
+                  {vulnerability.source === 'lookingGlass' && (
```

Comment:
@DJensen94 can you add a screenshot in the PR description that shows how the lookingGlass table looks like?